src/Inspetor/Model/InspetorSale.php

```diff
@@ -141,6 +137,10 @@ public function isValidUpdate() {
 			$this->validateStatus();
 		}
 
+		if (!$this->is_fraud) {
```

Comment:
mmm what happens here if `is_fraud == False`?

Reply:
Yeah! We should do `if (this->is_fraud == null)`

Reply:
That's right. I saw that and "fixed it" in the python lib creation, but didn't fix here. NOOOOOICE.